src/linearmixedmodel.jl

```diff
@@ -425,13 +426,13 @@ the length of `v` can be the rank of `X` or the number of columns of `X`.  In th
 case the calculated coefficients are padded with -0.0 out to the number of columns.
 """
 function fixef!(v::AbstractVector{T}, m::LinearMixedModel{T}) where {T}
-    Xtrm = first(m.feterms)
+    Xtrm = m.allterms[m.dims.nretrms + 1]
```

Comment:
why change this now?

(just curious)

Reply:
Again, I am thinking in terms of removing the `reterms` and `feterms` fields of `LinearMixedModel` entirely in favor of the principal that the `allterms` field always consists of a vector of `AbstractReMat{T}` terms followed by exactly one `XyMat` term.  There may be a separate field with a single FeMat like the current type (w/o the `wtx` field) to be able to handle rank-deficient cases.

So here I was planning that the expression would eventually be replaced by `last(m.allterms)`.